src/Pharmacy/Services/Import.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace OpenEMR\Pharmacy\Service;
+
+use Address;
+use OpenEMR\Common\Http\oeHttp;
+use Pharmacy;
+
+/**
+ * Class Import
+ * @package Import
+ * This class extends the Pharmacy class to import pharmacies listed with CMS.
+ * It can be adapted to work in other countries if a similar API is available.
+ * There is a duplication check using the NPI number. If the NPI number exist in the table. The entry is skipped.
+ * However, if the pharmacy gets a new NPI number, there could be two entries with the same address and different
+ * NPI numbers. I have discovered that some times erroneous entries can be retrieved.
+ */
+class Import extends Pharmacy
+{
+    /**
```

Comment:
Placeholder for ongoing review.

Reply:
I have removed the extends and will be testing all of the changes before uploading.

Reply:
Your code should work just fine without the extends.